Tests.Integration.Presentation.Web/Tools/Internal/UI_Configuration/UIConfigurationHelper.cs

```diff
@@ -3,37 +3,84 @@
 using System.Linq;
 using System.Net;
 using System.Net.Http;
-using System.Text;
 using System.Threading.Tasks;
 using Core.DomainModel.Organization;
-using Presentation.Web.Models.API.V1.GDPR;
 using Presentation.Web.Models.API.V1.UI_Configuration;
+using Tests.Integration.Presentation.Web.UI_Configuration;
 using Xunit;
 
 namespace Tests.Integration.Presentation.Web.Tools.Internal.UI_Configuration
 {
     public class UIConfigurationHelper
     {
+        private static readonly Random _random = new Random(DateTime.Now.Millisecond);
 
         public static async Task<HttpResponseMessage> SendPutRequestAsync(int organizationId, string module, UIModuleCustomizationDTO body, Cookie optionalLogin = null)
         {
-            var cookie = optionalLogin ?? await HttpApi.GetCookieAsync(OrganizationRole.GlobalAdmin);
-            var uri = TestEnvironment.CreateUrl($"api/v1/organizations/{organizationId}/ui-config/modules/{module}");
+            var cookie = optionalLogin ?? await HttpApi.GetCookieAsync(OrganizationRole.LocalAdmin);
             return await HttpApi.PutWithCookieAsync(TestEnvironment.CreateUrl($"api/v1/organizations/{organizationId}/ui-config/modules/{module}"), cookie, body);
         }
 
-        public static async Task<UIModuleCustomizationDTO> GetAllAsync(int organizationId, string module, Cookie optionalLogin = null)
+        public static async Task<UIModuleCustomizationDTO> GetCustomizationByModuleAsync(int organizationId, string module)
         {
-            using var response = await SendGetRequestAsync(organizationId, module, optionalLogin);
+            using var response = await SendGetRequestAsync(organizationId, module);
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
             return await response.ReadResponseBodyAsKitosApiResponseAsync<UIModuleCustomizationDTO>();
         }
 
+        public static async Task<UIModuleCustomizationDTO> CreateUIModuleAndSaveAsync(int organizationId, string module,
+            Cookie optionalLogin = null)
+        {
+            var dto = PrepareTestUiModuleCustomizationDto();
+            using var response = await SendPutRequestAsync(organizationId, module, dto, optionalLogin);
+
+            Assert.Equal(HttpStatusCode.NoContent, response.StatusCode);
+            return dto;
+        }
+
         public static async Task<HttpResponseMessage> SendGetRequestAsync(int organizationId, string module, Cookie optionalLogin = null)
         {
-            var cookie = optionalLogin ?? await HttpApi.GetCookieAsync(OrganizationRole.GlobalAdmin);
+            var cookie = optionalLogin ?? await HttpApi.GetCookieAsync(OrganizationRole.LocalAdmin);
```

Comment:
note that fallback will fail since the "default" cookies belong to the default org. That's why it used global admin which usually has all permissions. In this case, dont use a fallback and make the login cookie a required parameter